Plugins/SpatialGDK/SpatialGDKEditorToolbar/Source/SpatialGDKEditorToolbar/Private/InteropCodeGenerator/TypeBindingGenerator.cpp

```diff
@@ -842,7 +842,8 @@ void GenerateFunction_GetBoundClass(FCodeWriter& SourceWriter, UClass* Class)
 	if (Class->ClassGeneratedBy)
 	{
 		// This is a blueprint class, so use Unreal's reflection to find UClass pointer at runtime.
-		SourceWriter.Printf("return FindObject<UClass>(ANY_PACKAGE, TEXT(\"%s\"));", *Class->GetName());
+		//SourceWriter.Printf("return FindObject<UClass>(ANY_PACKAGE, TEXT(\"%s\"));", *Class->GetName());
+		SourceWriter.Printf("return LoadObject<UClass>(nullptr, TEXT(\"%s\"), nullptr, LOAD_None, nullptr);", *Class->GetPathName());
```

Comment:
I'm mildly concerned about what this implies for runtime memory/performance, but not sure if there is another way.

Maybe we can register interop only when we actually start to replicate an object, in which case it should certainly be loaded. But I feel like we went down that path before and there was a blocker.

Reply:
Yes agree it is not ideal, but I think we should see how big of a problem this is before we go down the route of optimising.